### PR TITLE
Added July update, updated some packages

### PR DESCRIPTION
**Pull request by duelinggalois**

### CODE REVIEW

File: docs/updates/July_2024.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+important catalyst for attention, providing momentum for the entire go-to-market
+campaign.
+
+We will have two X spaces scheduled
+
+- Monad Monday X Space - 11AM EST, August 26th
+- SYS AMA - Wednesday 10AM EST, August 28th
+
+Last minute we came across a partnership opportunity with PG group that led to a
+last minute decision to head to Token2049. I will share more details on how this
```
Comment (Uhan19) [inline docs/updates/July_2024.md:56]:
```suggestion
 decision to head to Token2049. I will share more details on how this
```
feels like we can drop the second `last minute` here.

**Review by Uhan19 — APPROVED** [review]

File: docs/updates/July_2024.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- Joining two spaces next month, one with Monad, dates and details below
+
+## Development
+
+Interest rate work is close to wrapping up and we are aiming to deploy a new
+version on testnet this month. We have a first draft of the liquidation
+protection that we are working through. The draft is still very early and it is
+going to take me some time to finalize. These are two last massive rocks that we
+have been working on for a while. Getting them done means that we are just
+fine-tuning things and crossing off smaller items. I don’t want to over promise
```
Comment (1m1-github) [inline docs/updates/July_2024.md:23]: "Getting them done means that we are just
fine-tuning things and crossing off smaller items." - this sentence is a bit unclear, getting the big rocks done means we will only have small things left to do, is what you mean -> "Getting them done means that we are left with just
fine-tuning things and crossing off smaller items."

File: docs/updates/July_2024.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+here, as it has been painful for me to watch everything take longer than I
```
Comment (1m1-github) [inline docs/updates/July_2024.md:24]: this negative prepend could be taken out, not sure how important it is

**Review by 1m1-github — APPROVED** [review]
2 comments, rest looks good

**Pull request merged**